packages/data/addon/models/navi-dimension.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import EmberObject from '@ember/object';
+import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
+
+export default class NaviDimensionModel extends EmberObject {
+  /**
+   * Dimension column definition
+   */
+  dimensionColumn!: DimensionColumn;
+
+  /**
+   * Dimension value
+   */
+  value!: unknown;
```

Comment:
For fili are you thinking something like `{ id: 1, description: 'dim description' }`?

Reply:
For Fili it might look like this: 

```ts
export default class FiliDimensionModel extends NaviDimensionModel {
  private rawValue = { id: 1, description: 'dim description' };

  get value() {
    return this.rawValue[this.dimensionColumn.parameters.field];
  }

  get displayValue() {
   return `${this.value} (${this.dimensionContext})`
  }
}
```